test.py

```diff
@@ -1109,6 +1112,7 @@ def _prepare_pytest_params(self, options: argparse.Namespace):
             "--run_id={}".format(self.id)
         ]
         self.args.append(f"--alluredir={self.allure_dir}")
+        self.args.append("--capture=tee-sys")
```

Comment:
Why is `--capture=tee-sys` needed here? Don't we need to specify `--allure-no-capture` here, like in the case of other types of tests?

Reply:
Yes, you are right.